src/commands/detail.js

```diff
@@ -123,7 +131,7 @@ module.exports = async function detail(flags) {
 		unactionable: flags.unactionable,
 	});
 
-	if (table) {
+	if (table & !flags.names) {
```

Comment:
```suggestion
	if (table) {
```
with the above change, we don't need to generate the table at all